pkg/minikube/extract/extract_test.go

```diff
@@ -79,3 +80,53 @@ func TestExtract(t *testing.T) {
 	}
 
 }
+
+func TestTranslationsUpToDate(t *testing.T) {
+	// Move the working dir to where we would run `make extract` from
+	cwd, err := os.Getwd()
+	if err != nil {
+		t.Fatalf("Getting current working dir: %v", err)
+	}
+
+	err = os.Chdir("../../..")
+	if err != nil {
+		t.Fatalf("Chdir failed: %v", err)
+	}
+	defer os.Chdir(cwd)
+
+	// The translation file we're going to check
+	exampleFile := "translations/fr-FR.json"
+	src, err := ioutil.ReadFile(exampleFile)
+	if err != nil {
+		t.Fatalf("Reading json file: %v", err)
+	}
+
+	// Create a temp file to run the extractor on
+	tempdir, err := ioutil.TempDir("", "temptestdata")
+	if err != nil {
+		t.Fatalf("Creating temp dir: %v", err)
+	}
+	defer os.RemoveAll(tempdir)
+
+	tempfile := filepath.Join(tempdir, "tmpdata.json")
+	err = ioutil.WriteFile(tempfile, src, 0666)
+	if err != nil {
+		t.Fatalf("Writing temp json file: %v", err)
+	}
+
+	// Run the extractor exactly how `make extract` would run, but on the temp file
+	err = TranslatableStrings([]string{"cmd", "pkg"}, []string{"translate.T"}, tempdir)
+	if err != nil {
+		t.Fatalf("Error translating strings: %v", err)
+	}
+
+	dest, err := ioutil.ReadFile(tempfile)
+	if err != nil {
+		t.Fatalf("Reading resulting json file: %v", err)
+	}
+
+	if bytes.Compare(src, dest) != 0 {
+		t.Fatalf("There are changes to the localizable strings in the code base.\n Please run `make extract`.")
```

Comment:
This should show a diff. `cmp.Diff` may be helpful here: I forget how it handles multi-line strings, but hopefully well.

```
if diff := cmp.Diff(src, dst); diff != "" {
  t.Errorf("Localized string mismatch (-want, +got):\n%s", diff)
}
```

Reply:
yep, cmp.Diff works as long as I unmarshal the file contents back to json so the result is human-readable. fixed.